src/main/webapp/app/app.component.html

```diff
@@ -2,14 +2,49 @@
   <mat-toolbar color="primary" class="mat-elevation-z4">
     <div class="wrapper">
       <div class="logo">
-        <img src="assets/images/logos/coyo/logo-coyo-inversed-nav-hd.png" alt="COYO">
+        <img
+          src="assets/images/logos/coyo/logo-coyo-inversed-nav-hd.png"
+          alt="COYO"
+        />
       </div>
-      <tiny-task-form (created)="created($event)"></tiny-task-form>
+      <tiny-task-form (created)="created()"></tiny-task-form>
     </div>
   </mat-toolbar>
-  <section class="content">
+  <section class="content flex-container">
+    <span style="display: flex"
+      ><h2>Click</h2>
+      <mat-icon class="title-icon">assignment</mat-icon>
+      <h2>to toggle status of Task</h2></span
```

Comment:
a poor UI decision in my opinion because a good UI would be intuitively understandable to the user without an explanation how to use it